resources/src/jvmMain/kotlin/dev/icerock/moko/resources/desc/ResourceFormattedStringDesc.kt

```diff
@@ -0,0 +1,11 @@
+package dev.icerock.moko.resources.desc
+
+import dev.icerock.moko.resources.StringResource
+
+actual class ResourceFormattedStringDesc actual constructor(
+    private val stringRes: StringResource,
+    private val args: List<Any>
```

Comment:
Right now all the android formatted strings we'll be replaced with `{n}` when we generate the `StringResource`, this means that we lose the type of the argument.

One possible solution would be to leave them as they are, and replace them prior to formatting (and get the necessary type information, before localizing them). See [MokoBundle](resources/src/jvmMain/kotlin/dev/icerock/moko/resources/desc/MokoBundle.kt)